src/Button.story.tsx

```diff
@@ -0,0 +1,309 @@
+/** @jsx jsx */
+import { jsx, css } from "@emotion/core";
+import React, { ComponentProps } from "react";
+import { storiesOf } from "@storybook/react";
+import { Button } from "./Button";
+// @ts-ignore We need to use `.tsx` because this will live next to the
+// transpiled files that we can't import.
+import { IconShip2 } from "../icons/IconShip2.tsx";
+import { colors } from ".";
+import * as typography from "./typography";
+
+const iconElement = <IconShip2 css={{ width: "100%", height: "100%" }} />;
+
+const DemoSection: React.FC<{
+  children: React.ReactNode;
+  description: string;
+  title: string;
+}> = ({ children, description, title, ...otherProps }) => (
+  <div {...otherProps} css={{ margin: 42 }}>
+    <div css={{ ...typography.base.xxlarge, marginBottom: ".5em" }}>
+      {title}
+    </div>
+    <div css={{ maxWidth: "50%", marginBottom: "5em" }}>{description}</div>
+    <div css={{ display: "flex", flexWrap: "wrap", margin: "0 -20px" }}>
+      {children}
+    </div>
+  </div>
+);
+
+/**
+ * Wrap children with a rounded border
+ */
+const ButtonWrapper: React.FC = ({ children, ...otherProps }) => (
+  <div
+    {...otherProps}
+    css={{
+      border: `1px solid ${colors.silver.dark}`,
+      borderRadius: 8,
+      margin: 6
+    }}
+  >
+    {children}
+  </div>
+);
+
+const VerticalButtonGroup: React.FC<{
+  buttonProps?: Partial<Omit<ComponentProps<typeof Button>, "children">>;
+  darkButtonProps?: Partial<Omit<ComponentProps<typeof Button>, "children">>;
+  children: JSX.Element | JSX.Element[];
+  title: string;
+}> = ({
+  buttonProps = {},
+  darkButtonProps = {},
+  children,
+  title,
+  ...otherProps
+}) => (
+  <div {...otherProps} css={{ margin: "0 20px", width: 300 }}>
+    <hr
```

Comment:
Why can't we use the existing typography component to style the titles of space kit stories?

Reply:
@mayakoneval 

Can you clarify what you mean? I don't understand because:

1. We don't have typography components; we only have constants defining the css properties
2. We _are_ using `typography.base` wherever we're defining typography styles

Reply:
oh yep, totally. tx!